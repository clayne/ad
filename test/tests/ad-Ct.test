ad | -C t | Waldo.txt | | 0
