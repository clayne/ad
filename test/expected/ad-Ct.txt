unsigned char Waldo_txt[] = {
  /* 0000000000000000 */ 0x57, 0x61, 0x6C, 0x64, 0x6F, 0x2E, 0x2E, 0x2E,
  /* 0000000000000008 */ 0x2E, 0x2E, 0x2E, 0x2E, 0x2E, 0x2E, 0x2E, 0x0A,
  /* 0000000000000010 */ 0x20, 0x57, 0x61, 0x6C, 0x64, 0x6F, 0x2E, 0x2E,
  /* 0000000000000018 */ 0x2E, 0x2E, 0x2E, 0x2E, 0x2E, 0x2E, 0x2E, 0x0A,
  /* 0000000000000020 */ 0x20, 0x20, 0x57, 0x61, 0x6C, 0x64, 0x6F, 0x2E,
  /* 0000000000000028 */ 0x2E, 0x2E, 0x2E, 0x2E, 0x2E, 0x2E, 0x2E, 0x0A,
  /* 0000000000000030 */ 0x20, 0x20, 0x20, 0x57, 0x61, 0x6C, 0x64, 0x6F,
  /* 0000000000000038 */ 0x2E, 0x2E, 0x2E, 0x2E, 0x2E, 0x2E, 0x2E, 0x0A,
  /* 0000000000000040 */ 0x20, 0x20, 0x20, 0x20, 0x57, 0x61, 0x6C, 0x64,
  /* 0000000000000048 */ 0x6F, 0x2E, 0x2E, 0x2E, 0x2E, 0x2E, 0x2E, 0x0A,
  /* 0000000000000050 */ 0x20, 0x20, 0x20, 0x20, 0x20, 0x57, 0x61, 0x6C,
  /* 0000000000000058 */ 0x64, 0x6F, 0x2E, 0x2E, 0x2E, 0x2E, 0x2E, 0x0A,
  /* 0000000000000060 */ 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x57, 0x61,
  /* 0000000000000068 */ 0x6C, 0x64, 0x6F, 0x2E, 0x2E, 0x2E, 0x2E, 0x0A,
  /* 0000000000000070 */ 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x57,
  /* 0000000000000078 */ 0x61, 0x6C, 0x64, 0x6F, 0x2E, 0x2E, 0x2E, 0x0A,
  /* 0000000000000080 */ 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20,
  /* 0000000000000088 */ 0x57, 0x61, 0x6C, 0x64, 0x6F, 0x2E, 0x2E, 0x0A,
  /* 0000000000000090 */ 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20,
  /* 0000000000000098 */ 0x20, 0x57, 0x61, 0x6C, 0x64, 0x6F, 0x2E, 0x0A,
  /* 00000000000000A0 */ 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20,
  /* 00000000000000A8 */ 0x20, 0x20, 0x57, 0x61, 0x6C, 0x64, 0x6F, 0x0A,
  /* 00000000000000B0 */ 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20,
  /* 00000000000000B8 */ 0x20, 0x20, 0x20, 0x57, 0x61, 0x6C, 0x64, 0x6F,
  /* 00000000000000C0 */ 0x0A, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20,
  /* 00000000000000C8 */ 0x20, 0x20, 0x20, 0x20, 0x57, 0x61, 0x6C, 0x64,
  /* 00000000000000D0 */ 0x6F, 0x0A, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20,
  /* 00000000000000D8 */ 0x20, 0x20, 0x20, 0x20, 0x20, 0x57, 0x61, 0x6C,
  /* 00000000000000E0 */ 0x64, 0x6F, 0x0A, 0x20, 0x20, 0x20, 0x20, 0x20,
  /* 00000000000000E8 */ 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x57, 0x61,
  /* 00000000000000F0 */ 0x6C, 0x64, 0x6F, 0x0A, 0x20, 0x20, 0x20, 0x20,
  /* 00000000000000F8 */ 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x57,
  /* 0000000000000100 */ 0x61, 0x6C, 0x64, 0x6F, 0x0A, 0x20, 0x20, 0x20,
  /* 0000000000000108 */ 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20,
  /* 0000000000000110 */ 0x57, 0x61, 0x6C, 0x64, 0x6F, 0x0A,
};
size_t Waldo_txt_len = 278;
